js/PivotReport.PivotTable.js

```diff
@@ -65,9 +68,9 @@ CRM.PivotReport.PivotTable = (function ($) {
 
     this.storeRecordFilterValues();
 
-    if (this.config.recordFilter) {
+    if (this.config.customFilter) {
```

Comment:
As this is renamed, we should change all other similar names
`initCustomFilterDefaultDateValues`
`customFilterForm`
etc